tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Framework.cs

```diff
@@ -87,11 +87,11 @@ internal SecurityCoordinator(Security security, HttpContext context, Span span)
 
             var messageContentProperty = messageType.GetProperty("Content");
             // StringContent(String content, Encoding, String mediaType)
-            var contentCtor = contentType.GetConstructor(new[] { typeof(string), typeof(Encoding), typeof(string) });
+            var contentCtor = contentType.GetConstructor(new[] { typeof(string), typeof(System.Text.Encoding), typeof(string) });
```

Comment:
Minor: but why add the namespace here, seems unnecessary.

Reply:
it was because at first I created a class named `Encoding `and it made conflicts, 🤦 but as I removed it later, it's not needed indeed, fixed in [fd8ae19](https://github.com/DataDog/dd-trace-dotnet/pull/4884/commits/fd8ae197a13170604645043af6b22181a7d3039d)